src/lib/scripts.js

```diff
@@ -104,6 +112,13 @@ const getScriptFieldValue = (contact, texter, fieldName) => {
 };
 
 export const applyScript = ({ script, contact, customFields, texter }) => {
+  // console.log("applyScript Method")
+  // console.log("SCRIPT from applyScriptMethod")
+  // console.log(script)
+  // console.log("CONTACT from applyScriptMethod")
+  // console.log(contact)
+  // console.log("ALLSCRIPTFIELDS from applyScriptMethod")
+  // console.log(scriptFields)
```

Comment:
This was my primary console.log until this code hits line 130 and the scriptFields is going through a getScriptFieldValue and breaking my code.

That's why all my console.logs live in the getScriptFieldValue on line 67.